{
  "dataType": "CVE_RECORD",
  "containers": {
    "cna": {
      "title": "Stellar-core's Overlay - security fix for DDoS mitigation",
      "source": {
        "advisory": "GHSA-mgx8-frjx-x33m",
        "discovery": "UNKNOWN"
      },
      "metrics": [
        {
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 5.9,
            "attackVector": "NETWORK",
            "baseSeverity": "MEDIUM",
            "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "integrityImpact": "NONE",
            "userInteraction": "NONE",
            "attackComplexity": "HIGH",
            "availabilityImpact": "HIGH",
            "privilegesRequired": "NONE",
            "confidentialityImpact": "NONE"
          }
        }
      ],
      "affected": [
        {
          "vendor": "stellar",
          "product": "stellar-core",
          "versions": [
            {
              "status": "affected",
              "version": "< 20.4.0"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://github.com/stellar/stellar-core/security/advisories/GHSA-mgx8-frjx-x33m",
          "name": "https://github.com/stellar/stellar-core/security/advisories/GHSA-mgx8-frjx-x33m",
          "tags": [
            "x_refsource_CONFIRM"
          ]
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "value": "Stellar-core is a reference implementation for the peer-to-peer agent that manages the Stellar network. Prior to 20.4.0, core nodes could be randomly crashed due to a race condition with a 3rd party library. The likelihood of affecting the network is low since crashed nodes come back up online right away. Code fix mitigation is part of Stellar-core v20.4.0 release"
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-362",
              "description": "CWE-362: Concurrent Execution using Shared Resource with Improper Synchronization ('Race Condition')"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
        "shortName": "GitHub_M",
        "dateUpdated": "2024-05-09T20:54:15.065Z"
      }
    },
    "adp": [
      {
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2024-32985",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-05-10T18:32:41.613694Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "shortName": "CISA-ADP",
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "dateUpdated": "2024-05-10T18:32:44.215Z"
        }
      }
    ]
  },
  "cveMetadata": {
    "cveId": "CVE-2024-32985",
    "state": "PUBLISHED",
    "dateUpdated": "2024-05-09T20:54:15.065Z",
    "dateReserved": "2024-04-22T15:14:59.167Z",
    "assignerOrgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
    "datePublished": "2024-05-09T20:54:15.065Z",
    "assignerShortName": "GitHub_M"
  },
  "dataVersion": "5.1"
}